STYLE.md

```diff
@@ -52,6 +52,27 @@ counts: Optional[Tuple[str, int]]
 ```
 
 
+### Use relative imports inside a package
+
+In code inside a package (directory), use relative `.` imports. For example, in the `ops` directory of the Ops codebase:
```

Comment:
```suggestion
When writing code inside a package (a directory containing an `__init__.py`), use relative imports with a `.` instead of absolute imports.
For example, within the `ops` package:
```